File: AAB1-PROYECTO/src/Propuesta_1.java
import java.util.Random;
import java.util.Scanner;
public class Propuesta_1 {
    public static void main(String[] args) throws Exception {
        Random random = new Random();
        Scanner dato= new Scanner(System.in);
        double noteACD, noteAPE, noteAA,total,percentApp,percentRep;
        String name,condition;
        int ced,contApp=0,contRep=0;
        do {
            System.out.println("Enter the name of the Student");
            name=dato.next();
            System.out.println("Enter the ID Student");
            ced=dato.nextInt();
            System.out.println("Bimonthly notes");
            noteACD=random.nextInt(10)+1;
            noteAPE=random.nextInt(10)+1;
            noteAA=random.nextInt(10)+1;
            total=(noteACD+noteAPE+noteAA)/3;
            System.out.println("Total Note: "+total);
            System.out.println("--------//REPORT//--------");
            System.out.println("--NOTES FOR BIMONTHLY--");
            System.out.println("Note ACD: "+noteACD);
            System.out.println("Note APE: "+noteAPE);
            System.out.println("Note AA: "+noteAA);
            if (total>=7) {
                System.out.println("Name Student: "+ name + " with ced: "+ced +" is APPROVED");
                contApp++;
            }else{
                System.out.println("Name Student: "+ name + "with ced: "+ced +" is NOT APPROVED");
                contRep++;
            }
            System.out.println("______________________________________");
            System.out.println("Do you want to enter one more student?");
            condition=dato.next().toLowerCase();
        } while (("s").equals(condition));
        percentApp=((contApp*100)/(contApp+contRep));
        percentRep=((contApp*100)/(contApp+contRep));
        System.out.println("______________________________________________");
        System.out.println("The percentage of students approved is: "+percentApp+"%");
        System.out.println("The percentage of students not approved is: "+percentRep+"%");
    }
}

File: AAB1-PROYECTO/PROPUESTA_1.py
import random
contApp=int (0)
contRep=int (0)
condition=str("s")
while condition== "s" or condition=="si":
    name = str (input ("Enter the name of the Student: ")) 
    ced = int (input ("Enter the ID Student: "))
    noteACD=random.randint(1,10)
    noteAPE=random.randint(1,10)
    noteAA=random.randint(1,10)
    total = float((noteAPE+noteAA+noteACD)/3) 
    print("--------//REPORT//--------")
    print("--NOTES FOR BIMONTHLY--");
    print("Note ACD: ",noteACD);
    print("Note APE: ",noteAPE);
    print("Note AA: ",noteAA);
    print("Total Note: ", total)
    if(total>=7):
        print("Student is approved for next semester.")
        contApp=contApp+1
    else:
        print("Student is not approved for next semester.")
        contRep=contRep+1
    print("______________________________________")
    condition = input("Do you want to enter one more student?").lower()
percentApp=float((contApp*100)/(contApp+contRep))
percentRep=float((contRep*100)/(contApp+contRep))
print("______________________________________________");
print("The percentage of students approved is: ",percentApp,"%");
print("The percentage of students not approved is: ",percentRep,"%");
